### DISCUSSION TITLE
Hello Everyone

### DISCUSSION THREAD

**Discussion by Cirhuzalain** [OpenMined/courses#123] [Introduce Yourself]
My name is Alain, I'm from the Democratic Republic of Congo, I'm really happy to be here to  wrap my head around various privacy preserving technologies and get my hands dirty in implementing them practically. I'm really looking forward to learning from you all.

**Comment by yemikifouly** [comment]
Hello Alain! 👋 Welcome to OpenMined and the discussion board! What field do you work in? It is always interesting to learn more about the background, interests and goals of our new members. 🙂 

Also, no matter your goal, I hope you enjoy the course very much. 🙂

**Reply by Cirhuzalain** [reply]
Hi Yemi,
I'm currently wrapping up with my master's degree in ML at the African Master's in Machine Intelligence (AMMI) and before that I was working as a Software Developer Consultant.

**Reply by yemikifouly** [reply]
How interesting! I have a couple of acquaintances doing a Master's degree in ML at the AMMI as well. Great program from what I hear. 🙂

**Reply by Cirhuzalain** [reply]
Indeed, it's really a nice program.
